src/main/app/case/definition.ts

```diff
@@ -469,7 +474,7 @@ export const enum PaymentStatus {
   CANCELLED = 'cancelled',
   ERROR = 'error',
 }
-export const CASE_TYPE = 'NO_FAULT_DIVORCE12';
+export const CASE_TYPE = 'NO_FAULT_DIVORCE14';
```

Comment:
this might break stuff as the LegalProcessed property has changed

Reply:
so the test might never pass as 2 of the properties were changed, 1 was a minor changed but the other requires abit of thought.

Reply:
Ah I needed to bump it to get the new jurisdiction field change

Reply:
yhh, if you can convert it go for it. Maybe i missed something when looking at it. Tom has a branch you can work off or copy.

Reply:
cool, I'll look into it. Worst case could we revert his changes for the time being?

Reply:
ideally, i would want that but then im not sure about the repercussion on the solicitor journey, we can also fake some of the data or pass null until its fixed but probably best to look at the ticket or discuss with Tom (hes away until tuesday)